overview.md

```diff
@@ -0,0 +1,35 @@
+# Jupyter Governance Overview
+
+Jupyter has undergone a transition from a Benevolent Dictator For Life (BDFL) + Steering Council model of governance to a new one, based on several related bodies. This document is a brief informational overview of this new model, which is being enacted as of October 2022.
+
+_Note:_ this document provides an informational summary of multiple documents in our governance repository that were individually voted on. Any deviation of , the underlying governance documents should be considered as the source of truth, and we will update this overview as needed.
```

Comment:
`Any deviation of , `
Looks like there's some words missing here

Reply:
Ah, yes - I was changing the wording on the original google doc here and we got distracted, I think. I'll fix this after merge, thx for catching it!